client/src/utils/data-formats/formats/timeline.ts

```diff
@@ -64,18 +64,21 @@ export function timeline(
   let values = state[args.data || 'values'];
   const timeline = values;
 
-  let _timeline = {};
+  let _timeline: { [time: number] : object} = {};
   let _lines = {};
 
   timeline.forEach(row => {
     let timestamp = row[timeField];
     let lineFieldValue = lineField === undefined ? valueField : row[lineField];
-    let valueFieldValue = row[valueField];
+    let valueFieldValue = parseFloat(row[valueField].toFixed(2));
```

Comment:
Either:
`let valueFieldValue = isNaN(row[valueField]) && row[valueField] || parseFloat(row[valueField].toFixed(2));`
Or:
`let valueFieldValue = parseFloat(row[valueField].toFixed(2)) || 0;` - Since NaN ==~ false